nntrainer/src/model_loader.cpp

```diff
@@ -50,8 +50,10 @@ int ModelLoader::loadModelConfigIni(dictionary *ini, NeuralNetwork &model) {
   model.loss_type = (LossType)parseType(
     iniparser_getstring(ini, "Model:Loss", unknown), TOKEN_LOSS);
   model.save_path = iniparser_getstring(ini, "Model:Save_path", "./model.bin");
-  model.batch_size =
-    iniparser_getint(ini, "Model:Batch_Size", model.batch_size);
+  if (model.isTrain()) {
```

Comment:
!model.isTrain()?

Reply:
no.. I guess 1 is enough for inference.

Reply:
batch_size will not be parsed if is not given anyway. As this code is gonna be frozen for the release, IMO we can let it parse unless there is a strict reason not to. And for dataset as well.

Reply:
for inference, we do not need batch_size.